content/download/lts/index.adoc

```diff
@@ -92,7 +92,8 @@ See the link:/content/event-calendar[event calendar] for the specific LTS RC/rel
 
 ## Backporting Process
 
-Any user can propose that a bug fix be backported to LTS by labeling with link:https://issues.jenkins.io/secure/IssueNavigator.jspa?reset=true&jqlQuery=labels+%3D+lts-candidate[lts-candidate].
+Any user can propose that a issue is backported to LTS by labeling with link:https://issues.jenkins.io/secure/IssueNavigator.jspa?reset=true&jqlQuery=labels+%3D+lts-candidate[lts-candidate].
+For an issue to be eligable for backporting it should be a small, safe-looking, uncontroversial fix for an important bug with no change to API surface, additionally library updates can also be considered if the library contains a public CVE that does not affect Jenkins (if Jenkins is vulnerable via a library then the link:/content/security[security] process applies).
```

Comment:
codifies tribal nature ( https://groups.google.com/g/jenkinsci-dev/c/3cZ15CiAFTQ/m/4NC52c_aBgAJ )  with the addition of non security security fixes to "shut the scanners up".  https://groups.google.com/g/jenkinsci-dev/c/jX0f6Kz6zhc